spec/lib/adv_ctc_irs1040_pdf_spec.rb

```diff
@@ -0,0 +1,255 @@
+require "rails_helper"
+
+require "rails_helper"
```

Comment:
`require`ing it once should be enough! ;)

Not a blocker, but would be a good cleanup.